test/fixtures/README

```diff
@@ -0,0 +1,2 @@
+Currently there is only one fixture and it's a symlink. This README serves as
```

Comment:
Can you extrapolate on what purpose the fixtures serve?

Reply:
They're files we need for testing. As-is, we don't actually store the one existing fixture because it's a symlink, which messes up Autotools, but that directory is where it goes.

Reply:
I would like to see an additional sentence here that just explains that the fixtures are for testing COPY in ch-grow.